sos/report/plugins/microshift.py

```diff
@@ -27,7 +27,7 @@ class Microshift(Plugin, RedHatPlugin):
     plugin_name = 'microshift'
     plugin_timeout = 900
     packages = ('microshift', 'microshift-selinux', 'microshift-networking',)
-    services = (plugin_name, 'microshift-etcd.scope',)
+    services = (plugin_name,)
```

Comment:
This will mean the plugin will *not* be automatically enabled when 'microshift-etcd.scope' service is present on the system - is that OK?